k8s/pkg/apis/longhorn/v1beta2/instancemanager.go

```diff
@@ -55,12 +55,13 @@ type InstanceProcessSpec struct {
 type InstanceState string
 
 const (
-	InstanceStateRunning  = InstanceState("running")
-	InstanceStateStopped  = InstanceState("stopped")
-	InstanceStateError    = InstanceState("error")
-	InstanceStateStarting = InstanceState("starting")
-	InstanceStateStopping = InstanceState("stopping")
-	InstanceStateUnknown  = InstanceState("unknown")
+	InstanceStateRunning    = InstanceState("running")
+	InstanceStateStopped    = InstanceState("stopped")
+	InstanceStateError      = InstanceState("error")
+	InstanceStateStarting   = InstanceState("starting")
+	InstanceStateStopping   = InstanceState("stopping")
+	InstanceStateUnknown    = InstanceState("unknown")
+	InstanceStateTerminated = InstanceState("terminated")
```

Comment:
Add a comment here.  IIUC, `terminated` means the instance is deleted and not found.

```suggestion
	InstanceStateTerminated = InstanceState("terminated")
```